landscape/client/package/tests/test_reporter.py

```diff
@@ -1220,6 +1221,17 @@ def test_resynchronize(self):
         self.facade.reload_channels()
         message_store = self.broker_service.message_store
         message_store.set_accepted_types(["package-locks"])
+
+        # create hash_id file
+        message_store.set_server_uuid("uuid")
+        self.facade.set_arch("arch")
+        hash_id_file = os.path.join(
+            self.config.hash_id_directory, "uuid_codename_arch")
+        os.makedirs(self.config.hash_id_directory)
+        with open(hash_id_file, "w") as f:
+            pass
+        self.assertFileContent(hash_id_file, b"")
```

Comment:
this assert seems a bit redundant, don't think we need to test behaviour of `open(..., 'w')`